examples/single_zone_cluster/README.md

```diff
@@ -1,3 +1,3 @@
 # Single Zone Cluster
 
-This is an example of creating a single zone VPC Gen2 OpenShift cluster
+This is an example of creating a single zone VPC Gen2 OpenShift cluster with cluster autoscaler addon.
```

Comment:
```suggestion
This example creates a VPC with a Red Hat OpenShift cluster in a single zone. The example includes autoscaling for the cluster.
```
Might not be exactly right, but we need to include Red Hat if we use OpenShift. I don't think gen2 is important, right?